Source/WebCore/svg/SVGAltGlyphDefElement.cpp

```diff
@@ -106,13 +106,15 @@ bool SVGAltGlyphDefElement::hasValidGlyphElements(Vector<String>& glyphNames) co
                 glyphNames.clear();
                 return false;
             }
-        } else if (!fountFirstGlyphRef && is<SVGAltGlyphItemElement>(child)) {
-            foundFirstAltGlyphItem = true;
+        } else if (!fountFirstGlyphRef) {
+            if (auto* AltGlyphItemElement = dynamicDowncast<SVGAltGlyphItemElement>(child)) {
+                foundFirstAltGlyphItem = true;
 
-            // As the spec says "The first 'altGlyphItem' in which all referenced glyphs
-            // are available is chosen."
-            if (downcast<SVGAltGlyphItemElement>(child).hasValidGlyphElements(glyphNames) && !glyphNames.isEmpty())
-                return true;
+                // As the spec says "The first 'altGlyphItem' in which all referenced glyphs
+                // are available is chosen."
+                if (AltGlyphItemElement->hasValidGlyphElements(glyphNames) && !glyphNames.isEmpty())
```

Comment:
Should not it be named `altGlyphItemElement`? The comment above is using 'altGlyphItem'.